docs/protocol.md

```diff
@@ -612,7 +565,12 @@ HTTP/1.1 200 OK
     {
       "id": "key1",
       "type": "Secp256k1VerificationKey2018",
```

Comment:
You probably don't want to use this... you probably want to use: https://github.com/w3c-ccg/lds-jws2020/blob/master/docs/example/didDoc.json#L18

because it covers all key types.

Reply:
Thanks Orie for bring this to our attention, I have pinged Daniel to further clarify and specify requirements on this in the Core spec, will implement the changes if any in a separate PR.